spec/folio_client_spec.rb

```diff
@@ -506,7 +526,6 @@
     before do
       stub_request(:post, "#{url}/authn/login")
         .to_return(
-          {status: 200, body: "{\"okapiToken\" : \"#{expired_token}\"}"},
```

Comment:
no need to simulate an initial eager login anymore, as the `before` block that applies to the whole spec now does the initial login.